pkg/templates/canal/canal.go

```diff
@@ -33,7 +33,7 @@ import (
 const (
 	installCNIImage = "calico/cni:v3.10.0"
 	calicoImage     = "calico/node:v3.10.0"
-	flannelImage    = "quay.io/coreos/flannel:v0.11.0"
+	flannelImage    = "quay.io/kubermatic/coreos_flannel:v0.11.0@sha256:3de983d62621898fe58ffd9537a4845c7112961a775efb205cab56e089e163b6"
```

Comment:
I'd prefer that we use the original image as default if that's possible and mirror only if that's not possible. Using the custom image is going to make some users suspicious (which is totally okay and understandable).

I'm aware that there is SHA256, but I'd still prefer if we use the original image if possible. :slightly_smiling_face:

Reply:
It's not possible to use originals, because there is no multi-arch manifest pushed to upstream.

Reply:
You are right. We don't have arch available in the API, so there is no way to check.